packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1195,6 +1206,19 @@ namespace Tpetra {
                     const Teuchos::ArrayView<const LocalOrdinal>& permuteToLIDs,
                     const Teuchos::ArrayView<const LocalOrdinal>& permuteFromLIDs) override;
 
+    void
+    resizeForIncomingDataImpl (const Kokkos::UnorderedMap<LocalOrdinal, size_t>& padding);
```

Comment:
This declaration uses the default `Kokkos::Device` for `UnorderedMap`.  I'm guessing that you would instead want to use the same Device that CrsGraph is using.  If so, you should specify the type as `Kokkos::UnorderedMap<LocalOrdinal, size_t, device_type>`.

Reply:
👍